client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -371,196 +399,274 @@ export default function CampaignItemDetails( props: Props ) {
 										</span>
-									</div>
+									</>
 								</div>
+
 								<div className="campaign-item-details__secondary-stats-row">
-									<div className="campaign-item-details__ad-destination">
-										<span className="campaign-item-details__label">
-											{ translate( 'Ad destination' ) }
-										</span>
-										<div className="campaign-item-details__ad-destination-url-container">
-											{ ! isLoading ? (
-												<Button
-													className="campaign-item-details__ad-destination-url-link"
-													href={ clickUrl }
-													target="_blank"
-												>
-													{ clickUrl }
-													{ getExternalLinkIcon() }
-												</Button>
-											) : (
-												<FlexibleSkeleton />
-											) }
+									<div className="campaign-item-details__secondary-stats-row-left">
+										<>
+											<span className="campaign-item-details__label">
+												{ translate( 'Languages' ) }
+											</span>
+											<span className="campaign-item-details__details">
+												{ ! isLoading ? languagesListFormatted : <FlexibleSkeleton /> }
+											</span>
+										</>
+									</div>
+									<div className="campaign-item-details__second-column">
+										<div className="campaign-item-details__second-column-languages">
+											<span className="campaign-item-details__label">
+												{ translate( 'Languages' ) }
+											</span>
+											<span className="campaign-item-details__details">
+												{ ! isLoading ? languagesListFormatted : <FlexibleSkeleton /> }
+											</span>
+										</div>
+										<div className="campaign-item-details__second-column-interests">
+											<span className="campaign-item-details__label">
+												{ translate( 'Interests' ) }
+											</span>
+											<span className="campaign-item-details__details">
+												{ ! isLoading ? topicsListFormatted : <FlexibleSkeleton /> }
+											</span>
+										</div>
+										<div>
+											<span className="campaign-item-details__label">
+												{ translate( 'Location' ) }
+											</span>
+											<span className="campaign-item-details__details campaign-item-details__locations">
+												{ ! isLoading ? (
+													<TargetLocations audienceList={ audience_list } />
+												) : (
+													<FlexibleSkeleton />
+												) }
+											</span>
+										</div>
+										<div className="campaign-item-details__ad-destination">
+											<span className="campaign-item-details__label">
+												{ translate( 'Destination' ) }
+											</span>
+											<div className="campaign-item-details__ad-destination-url-container">
+												{ ! isLoading ? (
+													<Button
+														className="campaign-item-details__ad-destination-url-link"
+														href={ clickUrl }
+														target="_blank"
+													>
+														{ isWooStore ? translate( 'Product page' ) : translate( 'Post page' ) }
```

Comment:
We don't have a way to differentiate between posts, pages, and products in here. Even in WooCommerce, you can still promote posts and pages.

We need to find a more neutral name until we change the backend to return the type of post.